### PR TITLE
adapt disconnect message

### PR DESCRIPTION
**Pull request by namiwang**
https://github.com/rails/rails/pull/34194/ has already been merged

### CODE REVIEW

**Comment by namiwang** [comment]
@palkan Hope that's ok

File: docs/misc/action_cable_protocol.md [hunk 1]
```diff
@@ -32,11 +33,9 @@ Possible commands:
 When client connects to server one of the following two could happen:
 
 * server accepts the connection and responds with `welcome` message (`{"type":"welcome"}`)
-* server rejects and drops the connection without sending anything\*
+* server rejects the connection and responds with a `disconnect` message, which may include fields `reason` and `reconnect` (`{"type":"disconnect", "reason":"unauthorized", "reconnect":false}`)
```
Comment (palkan) [inline docs/misc/action_cable_protocol.md]: Let's add a note that it has been added in Rails 6.0

**Review by palkan — APPROVED** [review]
Thanks!
Just one minor comment

**Review by palkan — APPROVED** [review]
Thanks!

**Pull request merged**